filebeat/docs/modules/haproxy.asciidoc

```diff
@@ -0,0 +1,62 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-haproxy]]
+:modulename: haproxy
+
+== haproxy module
+
+The +{modulename}+ module collects and parses logs from a (`haproxy`) process.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+The +{modulename}+ module was tested with logs from `haproxy` running on AWS Linux as a gateway to a cluster of microservices.
+
+This module is not available for Windows.
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboard
+
+This module comes with a sample dashboard showing geolocation, distribution of requests between backends and frontends,
+and status codes over time. For example:
+
+[role="screenshot"]
+image::./images/kibana-haproxy-overview.png[]
+
+include::../include/configuring-intro.asciidoc[]
+
+The module is by default configured to run via syslog on port 9001. However
+it can also be configured to read from a file path. See the following example.
```

Comment:
Nice addition to be able to use syslog and file.

Reply:
Thanks. I started out with a normal file log but setting it up in HAProxy requires a bit of effort. Going via syslog was easier as HAProxy has that support built in. I intend to document the HAProxy configuration necessary for using the module too.